packages/http-client-csharp/scripts/add-debug-profile.js

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env node
+
+/**
+ * Script to add launch settings profile for easy debugging of TypeSpec generation
+ * 
+ * This script:
+ * 1. Installs tsp-client if not already installed
+ * 2. Runs tsp-client sync in the target SDK directory
+ * 3. Runs tsp-client generate --save-inputs to create tspCodeModel.json
+ * 4. Adds a new debug profile to launchSettings.json that targets the DLL
+ */
+
+import { execSync, spawn } from 'child_process';
```

Comment:
No - this should be a powershell script and it should go under the eng/scripts folder within http-client-csharp.